src/pages/debrief/cat-a-mod1/components/speed-check-debrief-card/speed-check-debrief-card.html

```diff
@@ -1,4 +1,4 @@
-<ion-card id="speed-check" *ngIf="emergencyStop.firstAttempt || avoidance.firstAttempt">
+<ion-card id="speed-check" *ngIf="emergencyStop.firstAttempt !== undefined || avoidance.firstAttempt !== undefined">
```

Comment:
use the same function that is implemented in the selector

Reply:
The selector checks if the avoidance was attempted, it doesn't check if the emergency stop was like above. There could be a function created if desired but the suggestion to use the selector wouldn't work.